src/ext.js

```diff
@@ -272,6 +272,71 @@ export default function ext({ translator }) {
                     },
                   },
                 },
+                fontStyling: {
+                  component: 'item-selection-list',
+                  type: 'string',
+                  ref: 'style.textStyle',
+                  translation: 'Text Style',
+                  horizontal: true,
+                  multipleSelect: true,
+                  items: [
+                    {
+                      component: 'icon-item',
+                      icon: 'bold',
+                      value: 'bold',
+                      translation: 'Bold',
```

Comment:
Strings needed

Reply:
Probably best to put in the translation strings before merging. Even if we don't add them to the client yet it would be easier to spot not translated strings

Reply:
sure I fix